guides/source/active_record_callbacks.md

```diff
@@ -340,7 +340,7 @@ end
 
 ### Combining Callback Conditions
 
-When multiple conditions define whether or not a callback should happen, an `Array` can be used. Moreover, you can apply both `:if` and `:unless` to the same callback.
+When multiple conditions define whether or not a callback should execute, an `Array` can be used. Moreover, you can apply both `:if` and `:unless` to the same callback.
```

Comment:
We can also say 'run' instead of 'execute' as we've used on [another line](https://github.com/rails/rails/pull/35324/files#diff-23bde35a6dadcaeb7fb6918f754ee591R353).